jobbergate-agent-snap/Makefile

```diff
@@ -16,16 +16,16 @@ format: ## Run the formatter
 	tox -e format
 
 .PHONY: build
-build: ## Build the snap
-	@git rev-parse --short HEAD > sha/commit-sha.txt
-	@snapcraft $(ARGS)
+build: clean ## Build the snap
+	@snapcraft -v
+	mv *.snap jobbergate-agent.snap
```

Comment:
Can you explain this part?

Reply:
Sure!

I made it dependent on `clean` so that the build environment would be fresh and any leftover `.snap` artifacts would be removed.

The command `snapcraft -v` builds the snap using verbose mode. (I can't imagine not wanting to see the output because if something goes wrong, you have to run it again in verbose mode anyway. Additionally, the CI was already using verbose mode. Just seemed like the right move.